### PR TITLE
Changes required for App sre

### PR DESCRIPTION
**Pull request by riprasad**

### CODE REVIEW

File: distro/docker/src/main/docker/Dockerfile.jvm [hunk 1]
```diff
@@ -1,9 +1,12 @@
-FROM fabric8/java-alpine-openjdk11-jre
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
 
 ENV APP_URL="${docker.app.file}"
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 ENV AB_ENABLED=jmx_exporter
+ENV JAVA_APP_DIR=/deployments
 
-ADD "$APP_URL" /deployments
+EXPOSE 8080 8778 9779
```
Comment (EricWittmann) [inline distro/docker/src/main/docker/Dockerfile.jvm:8]: What are the 8778 and 9779 ports used for?

Comment (riprasad) [reply distro/docker/src/main/docker/Dockerfile.jvm:8]: umm I am not sure! I used the dockerfile we use for productization and I simply copied it

**Comment by famartinrh** [comment]
great job!

File: distro/docker/src/main/docker/Dockerfile.jvm (hunk 1 above)
Comment (EricWittmann) [reply distro/docker/src/main/docker/Dockerfile.jvm:8]: OK we can keep it as-is for now.  

@Apicurio/developers any idea what those two additional ports are for?  I'm thinking 8778 might be the debug port.  But I have no idea what 9779 would be.

**Review by EricWittmann — APPROVED** [review]

**Pull request merged**